third_party/terraform/tests/resource_notebooks_instance_test.go.erb

```diff
@@ -0,0 +1,77 @@
+<% autogen_exception -%>
+package google
+
+<% unless version == 'ga' %>
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccNotebooksInstance_update(t *testing.T) {
+	t.Parallel()
+
+	prefix := fmt.Sprintf("%d", randInt(t))
+	name := fmt.Sprintf("tf-test-%s", prefix)
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckNotebooksInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNotebooksInstance_update(prefix, name),
+			},
+			{
+				ResourceName:      "google_notebooks_instance.test",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAccNotebooksInstance_update_gpu(prefix, name),
+			},
+			{
+				ResourceName:      "google_notebooks_instance.test",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccNotebooksInstance_update(prefix, name string) string {
+	return fmt.Sprintf(`
+
+resource "google_notebooks_instance" "test" {
+  name = "%s"
```

Comment:
This call has only one `%s` but accepts two arguments - this upsets the linter, so if you can fix it it'll be a little nicer.  :)

Reply:
ack, I fixed, but Build still failing, (try to get access to logs but I don't have permission) any pointers will be great thanks